modules/olm-updating-index-image.adoc

```diff
@@ -1,43 +1,45 @@
 // Module included in the following assemblies:
 //
 // * operators/olm-managing-custom-catalogs.adoc
+// * operators/olm-restricted-network.adoc
 
 [id="olm-updating-index-image_{context}"]
 = Updating an index image
 
-You can update an existing index image using the `opm` CLI.
+After configuring OperatorHub to use a CatalogSource that references a custom index image, cluster administrators can keep the available Operators on their cluster up to date by adding bundle images to the index image.
+
+You can update an existing index image using the `opm index add` commad.
 
 .Prerequisites
 
 * `opm` version 1.12.3+
 * `podman` version 1.4.4+
 * An index image built and pushed to a registry.
-* A CatalogSource created and applied to a cluster.
+* An existing CatalogSource referencing the index image.
 
 .Procedure
 
-. Update the existing index:
+. Update the existing index by adding bundle images:
 +
 [source,terminal]
 ----
 $ opm index add \
-    --bundles quay.io/<namespace>/another-operator:v1 \//<1>
-    --from-index quay.io/<namespace>/test-catalog:latest \//<2>
-    --tag quay.io/<namespace>/test-catalog:latest <3>
+    --bundles <registry>/<namespace>/<new_bundle_image>:<tag> \//<1>
+    --from-index <registry>/<namespace>/<existing_index_image>:<tag> \//<2>
+    --tag <registry>/<namespace>/<index_image>:<tag> <3>
 ----
-<1> The additional bundle image to add to the index.
+<1> A comma-separated list of additional bundle images to add to the index.
 <2> The existing index that was previously pushed.
 <3> The image tag that you want the updated index image to have.
 
 . Push the updated index image:
 +
 [source,terminal]
 ----
-$ podman push quay.io/<namespace>/test-catalog:latest
+$ podman push <registry>/<namespace>/<index_image>:<tag>
```

Comment:
@adellape I suggest to change it to "podman push <registry>/<namespace>/<existing_index_image>:<tag>"